src/main/java/seedu/partyplanet/model/event/Event.java

```diff
@@ -0,0 +1,135 @@
+package seedu.partyplanet.model.event;
+
+import seedu.partyplanet.model.person.Birthday;
+import seedu.partyplanet.model.person.Name;
+import seedu.partyplanet.model.person.Remark;
+
+/**
+ * Represents a Event in PartyPlanet.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class Event {
+
+    // Identity fields
+    private Name name;
+
+    // Data fields
+    private Birthday date;
+    private Remark details;
+    private boolean isDone;
+
+    /**
+     * Default Event constructor
+     */
+    public Event(Name name, Birthday date, Remark details) {
+        this.name = name;
+        this.date = date;
+        this.details = details;
+        this.isDone = false;
+    }
+
+    /**
+     * Event constructor with a field for isDone
+     */
+    public Event(Name name, Birthday date, Remark details, boolean isDone) {
+        this.name = name;
+        this.date = date;
+        this.details = details;
+        this.isDone = isDone;
+    }
+
+
+    /**
+     * Returns name of event
+     */
+    public Name getName() {
+        return name;
+    }
+
+    /**
+     * Returns date of event
+     */
+    public Birthday getDate() {
+        return date;
+    }
+
+    /**
+     * Returns details of event
+     */
+    public Remark getDetails() {
+        return details;
+    }
+
+    /**
+     * Returns whether Event is done.
+     */
+    public boolean isDone() {
+        return isDone;
+    }
+
+    /**
+     * Returns event status.
+     * Tick represents done, cross represents not done.
+     */
+    public String getStatus() {
+        return isDone ? "\u2713" : "";
+    }
```

Comment:
Is the else supposed to return a cross instead of empty string? If not maybe change the javadoc accordingly.

Reply:
No, we decided that the cross for an undone Event would look very unappealing, and so we've removed it

Reply:
Will update the javadocs accordingly, thanks!

Reply:
Ok!